allennlp/tests/commands/find_learning_rate_test.py

```diff
@@ -178,3 +180,82 @@ def test_search_learning_rate_without_stopping_factor(self):
         learning_rates, losses = search_learning_rate(self.trainer, num_batches=100, stopping_factor=None)
         assert len(learning_rates) == 101
         assert len(losses) == 101
+
+class TestFindLearningRateForMultiGPU(AllenNlpTestCase):
```

Comment:
Instead of making a new class for this, just modify the class above to save `params` to `self`, and change `params['trainer']['cuda_device']` in your test.